Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -1485,24 +1482,42 @@ public void SwapForSameTip(bool render)
         }
 
         [Theory]
-        [InlineData(true, true)]
-        [InlineData(true, false)]
-        [InlineData(false, true)]
-        [InlineData(false, false)]
-        public async Task SwapWithoutReorg(bool direction, bool render)
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task SwapWithoutReorg(bool render)
         {
             BlockChain<DumbAction> fork = _blockChain.Fork(_blockChain.Tip.Hash);
 
-            // When direction == true:  the higher chain goes to the lower  chain  [#N -> #N-1]
-            // When direction == false: the lower  chain goes to the higher chain  [#N -> #N+1]
-            await (direction ? _blockChain : fork).MineBlock(default);
-            var prevRecords = _renderer.ReorgRecords;
+            // The lower  chain goes to the higher chain  [#N -> #N+1]
```

Comment:
```suggestion
            // The lower chain goes to the higher chain  [#N -> #N+1]
```

It seems just useless space not affecting its functions. If it makes your task confused, please ignore this.